vignettes/config_yml.Rmd

```diff
@@ -0,0 +1,479 @@
+
+```yaml
+    scenario_select: "nze_2050"
+```
+
+#### region_select
+
+`region_select`{.yaml} is an identifier of the region to be used. It must be a single string/character value, and it must refer to a valid, accessible, region identifier contained in the [`r2dii.data::region_isos`{.r}](https://rmi-pacta.github.io/r2dii.data/reference/region_isos_demo.html) datatset. Valid values typically look like `"global"`{.yaml} or `"advanced economies"`{.yaml}. As an example:
+
+```yaml
+    region_select: "global"
+```
+
+#### start_year
+
+`start_year`{.yaml} is the start year of the analysis. Normally, the start year should correspond with year of the publication of the scenario in use. It must be a single numeric value, and it must refer to a valid, accessible, year contained in the scenario data file/s defined by `filename_scenario_tms`{.yaml} and `filename_scenario_sda`{.yaml}. Valid values typically look like `2022`{.yaml} or `2023`{.yaml} (note that this value should *not* be wrapped in quotes). As an example:
+
+```yaml
+    start_year: 2022
+```
+
+#### time_frame
+
+`time_frame`{.yaml} is the number of years (starting from the `start_year`{.yaml}) that the analysis covers, defining the time frame. It must be a single numeric value, and it must define a valid, accessible, time frame  covered by the scenario data file/s defined by `filename_scenario_tms`{.yaml} and `filename_scenario_sda`{.yaml}. Valid values typically look like `5`{.yaml} or `6`{.yaml} (note that this value should *not* be wrapped in quotes). As an example:
+
+```yaml
+    time_frame: 5
+```
+
+#### benchmark_regions_select
+
+`benchmark_regions_select`{.yaml}. As an example:
+
+```yaml
+    # regions must be available for the selected scenario
+    benchmark_regions_select: "global,european union"
+```
+
+### sector_split:
+
+A full example `sector_split`{.yaml} section might look like:
+
+```yaml
+  sector_split:
+    apply_sector_split: TRUE
+    sector_split_type: "equal_weights"
+    dir_split_company_id: "~/Desktop/test/split"
+    filename_split_company_id: "split_company_ids.csv"
+    dir_advanced_company_indicators: "~/data/pactarawdata/asset-impact/2024-02-15_AI_RMI_2023Q4"
+    filename_advanced_company_indicators: "2024-02-14_AI_2023Q4_RMI-Company-Indicators.xlsx"
+    sheet_advanced_company_indicators: "Company Activities"
+```
+
+#### apply_sector_split
+
+`apply_sector_split`{.yaml} It must be a single logical value (either `TRUE`{.yaml} or `FALSE`{.yaml}). As an example:
+
+```yaml
+    apply_sector_split: TRUE
+```
+
+#### sector_split_type
+
+`sector_split_type`{.yaml}. As an example:
+
+```yaml
+    sector_split_type: "equal_weights"
+    # sector_split_type: "worst_case"
```

Comment:
```suggestion
```

Reply:
`"worst_case"` is not supported anymore, it was a weird rule. At the moment, `"equal_weights"` is the only option, I just did not remove that layer because it would seem like a good idea to provide more options if the sector split functionality is to be maintained long term